README.md

```diff
@@ -42,11 +49,29 @@ let dog = new Dog("Shadow");
 dog.speak(); // Shadow says woof!
 ```
 
+( KG: Code like this needs a post-game explanation of what it is doing.)
```

Comment:
@drakeltheryuujin Generally, it's OK to include some English after a hunk of code drops.  In the original you had it (effectively) _after_ the super method. It probably needed to be bumped up here.